### ISSUE TITLE
ExtendedLoadout compatibility suggestion

### ISSUE THREAD

**Issue by aaronfreed** [DestroyedClone/AncientScepter#45]
If you’re also running ExtendedLoadout, a second Scepter shouldn’t automatically reroll into a random red item if it can instead upgrade a second skill. Right now, the only way I know to upgrade two skills is to first pick up a Scepter from ClassicItems, then pick up the standalone scepter (which requires editing the StandaloneAncientScepter config, at that). Kinda silly.

**Comment by astroeblaze** [comment]
I definitely would like to see some better compatibility with Ancient Scepter and Extended Loadout if it is at all possible

**Comment by aaronfreed** [comment]
Agreed. And now that I think about it, a config to set which skills take priority if multiple skills can be upgraded might also be nice (maybe even per-character configs, as long as we’re getting into “stretch goal” territory).
